styleguide/derek/_partials/grid/widget/blog-feed-item.ejs

```diff
@@ -0,0 +1,23 @@
+<!-- Blog Feed Pattern -->
+
+<div class="blogFeed-item">
+  <div class="row">
```

Comment:
Do we need to do that if there is no special css, js or classing there at all? Wouldnt it just be adding a class for no reason?